src/gui/Src/Utils/StringUtil.cpp

```diff
@@ -224,4 +226,23 @@ QString ToDateString(const QDate & date)
         "Dec"
     };
     return QString().sprintf("%s %d %d", months[date.month() - 1], date.day(), date.year());
+    */
+    return QLocale(QString(currentLocale)).toString(date);

```

Comment:
Yeah then use locale dependent dates in the thread creation 👍
On Tue, 26 Jul 2016 at 04:33, Torusrxxx <notifications@github.com> wrote:

> In src/gui/Src/Utils/StringUtil.cpp
> <https://github.com/x64dbg/x64dbg/pull/885#discussion_r72179404>:
>
> > @@ -224,4 +226,23 @@ QString ToDateString(const QDate & date)
> >          "Dec"
> >      };
> >      return QString().sprintf("%s %d %d", months[date.month() - 1], date.day(), date.year());
> > +    */
> > +    return QLocale(QString(currentLocale)).toString(date);
>
> In fact it is also used in the thread creation time, that's why I want
> locale-dependent date formatting.
>
> —
> You are receiving this because you modified the open/close state.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/x64dbg/x64dbg/pull/885/files/bd8ac511d5e4a299efb5d3d766f9dbe3aea7f7ed#r72179404>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACWCmeX4S9bP5z-TdMbQ87A_jabnyY-Iks5qZXHzgaJpZM4JUIpX>
> .
>